### ISSUE TITLE
Foundry Chat Entries That Include BULLETS Difficult to Read

### ISSUE THREAD

**Issue by AndreCaseyScott** [therealguy90/foundrytodiscord#9]
**Describe the bug**
The formatting of chat log entries that include BULLETS does not mirror via the webhook appropriately make the Discord version hard to read. 

Before answering the questions on this template, are you on the latest release of this module? YES
- Is the bug caused by a conflict with this module? **NO, tried "Find the Culprit" with just this module**
- Is this module throwing an error? **NO, seems to work.**
- Is the message from the webhook not like you expected? **YES, formatting is not retained in a readable form.** 

**To Reproduce**
1. Go to: Any class feature that includes BULLET POINTS. E.g. Arms of the Astral Self
2. Click on: Click to activate feature, sending text of skill to chat. 
3. Check Discord chat posting of that feature

**Expected behavior**
I expected the bullets in Discord to at least have a hypen ("-") to delineate the items, similar to how they appear in the Foundry chat log. I'm not expecting the SAME appearance, but some form of separation, especially since Discord supports markdown. 

**Screenshots**
Snippet of Chat Log of Feature - 
![Screen Shot 2023-09-03 at 17 52 39](https://github.com/therealguy90/foundrytodiscord/assets/3774345/f85ad412-b180-45ca-8a9b-726d46a9b980)

Discord log of mirrored feature - 
![Screen Shot 2023-09-03 at 17 52 43](https://github.com/therealguy90/foundrytodiscord/assets/3774345/ca0ab740-6724-4fc2-972a-64da0e157866)

**Comment by therealguy90** [comment]
Re-tagged all three issues #10, #11 as feature requests and not bugs. This is the current intended behavior. I understand readability is important, but next time, instead of making three separate issues, make one feature request with all three as 'accessibility enhancements'.

**Comment by AndreCaseyScott** [comment]
I appreciate the follow up. I didn't see any other tags available when I was posting the issues and figured you'd be able to reassign, as needed. 

I opted to enter them separately as I imagined that they required different levels of difficulty to solve. Bullets would be an easy hypen addition where I imagine tables will be much harder to parse. 

Either way, I'll keep your request in mind for future issues.
